src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -71,6 +75,66 @@ public void RunBenchmarks(string assemblyPath)
             s_profiler(assemblyPath, Configuration.RunId, _outputDirectory, runner, collectOutputFilesCallback);
         }
 
+        public void RunStartupBenchmark(string[] args, ProcessStartInfo psi, PreDel pre, MidDel mid, PostDel post)
```

Comment:
>PreDel pre, MidDel mid, PostDel post [](start = 77, length = 36)

I would suggest replacing the delegates with Actions<PerformanceTestConfig> (this function can become generic in the future too).